spaceship/lib/spaceship/two_step_client.rb

```diff
@@ -131,36 +83,17 @@ def select_device(r, device_id)
       code = ask("Please enter the 4 digit code: ")
       puts "Requesting session..."
 
-      # Send token back to server to get a valid session
+      # Send code back to server to get a valid session
       r = request(:post) do |req|
-        req.url "https://idmsa.apple.com/appleauth/auth/verify/device/#{device_id}/securitycode"
-        req.body = { "code" => code.to_s }.to_json
-        req.headers['Content-Type'] = 'application/json'
-
-        update_request_headers(req)
+        req.url "https://idmsa.apple.com/IDMSWebAuth/validateSecurityCode"
+        req.body = "digit1=#{code[0]}&digit2=#{code[1]}&digit3=#{code[2]}&digit4=#{code[3]}&scnt=#{scnt}"
```

Comment:
why 🍎, would go away from a JSON post
kek https://github.com/amyworrall/QuickRadar/blob/8326a52ba49f47c1faad59117ead6935987f91e0/QuickRadar/QRRadarSubmissionService.m#L247
https://github.com/clayfreeman/IDMSWebAuth/blob/f5e3a6ed8be4bc76f53f7280209faaaa8d40823d/include/IDMSWebAuth.php#L147
seems like you should handle between 4 or 6 digit.

Could we do some kind of array and then just join on '&'